index.csv

```diff
@@ -189,6 +193,7 @@ repeat-element,MIT,1.1.2
 request,Apache-2.0,2.34.0,2.35.0,2.36.0
 require-all,MIT,0.0.6,0.0.8,1.0.0
 require-directory,MIT,0.0.1,0.0.2,0.0.3,1.0.0,1.0.1,1.0.2,1.1.0,1.1.1,1.2.0,2.0.0,2.1.0
+require-like,MIT,0.0.1,0.0.2,0.1.0,0.1.1,0.1.2
```

Comment:
Looks good. Author added license file in initial commit: https://github.com/felixge/node-require-like/commit/00d9b7a81bec66aa0aab2530073ff2ba976caf20